docs/translations.md

```diff
@@ -1,3 +1,57 @@
+# Managing strings
+
+We use Android's [resource system](https://developer.android.com/guide/topics/resources/localization) to localize
+user-visible strings in our apps.
+
+Our source language is English (American English to be more precise, but simply "English" (en) on Weblate).
+
+Feature modules are expected to store strings exclusively used by them in their `res/values/strings.xml` file.
+There's currently no final place for strings shared between feature modules. For now there's some in
+[:legacy:ui:base](/legacy/ui/base/src/main/res/values/strings.xml).
```

Comment:
I think this lacks guidance for shared strings, and it’s worth questioning whether shared strings are a good idea at all. 

I see that shared strings promise to be easier maintainable (one update applies in all places), reduce duplication and smaller file sizes. However, they come with significant challenges:
- Strings are often context-dependent, and using them across multiple contexts can lead to awkward or incorrect messaging. 
- A shared module introduces dependencies between modules, complicating the build process and reducing modularity.
- Developers may overuse shared strings to save effort, even when it’s not appropriate.
- Shared strings can confuse translators if the intended use or context is not well-documented, increasing the risk of mistranslations.
- Modifying shared strings can have unintended consequences, as the change might not align with all the contexts in which it is used. This cascades down to translations.
- Introducing shared strings can lead to prolonged debates among team members about whether a string qualifies as "shared."

We should find a clear definition for shared strings. From my point they should be limited to truly universaI strings, such as `Save`, `Cancel` or branding-related messages. Additionally:
- They should provide context to document the intended use of each shared string, helping developers and translators to understand its purpose.
- We should provide a long term solution instead of advertising the old legacy module. A better approach could be to create a dedicated shared resource module.
- Create shared string collections for specific domains (e.g., "Mail", "Authentication") rather than a single global resource.
- Define clear ownership of shared strings. Who decides what gets added to the shared resource? 
- Review shared strings on regular basis to check their relevance and if they are appropriately used.

This should help to minimize subjective discussions and keep the workflow efficient. However, it's essential to carefully evaluate whether the benfits of shared strings outweight these challenges in the first place.

Reply:
I agree with most of this. Except for:

> - Define clear ownership of shared strings. Who decides what gets added to the shared resource?
> - Review shared strings on regular basis to check their relevance and if they are appropriately used.

This feels very heavy-handed and a more rigorous process than we apply to our code.

I think it's best to have that discussion in a separate pull request. I'm happy to remove the last sentence about shared strings or the whole paragraph for now, so that we can get the rest of the document merged.

Reply:
Then let's remove the sentence.